openedx/course-discovery
# Django 3.2 upgrade: replace drf dynamic fields

Related ticket at https://github.com/edx/upgrades/issues/49.

**Testing instructions:**
1. Make sure all CI checks are passing.
2. Checkout this branch on your devstack.
3. Run `make discovery-shell` and install requirements `make requirements`.
4. Log in the LMS, then go to the discovery courses API and test a few cases:
- Check the `omit` parameter works: http://localhost:18381/api/v1/courses/?omit=title
- Check the `fields` parameter works: http://localhost:18381/api/v1/courses/?fields=title

*Reviewers:*
- [ ] @awais786
